src/SumoCoders/FrameworkCoreBundle/Resources/assets/js/Framework/Form.js

```diff
@@ -92,64 +96,25 @@ export class Form {
   }
 
   dateFields () {
-    if (!$.isFunction($.fn.datepicker)) {
-      return
-    }
-
-    $.datepicker.setDefaults(dateFieldOptions)
-
-    $('[data-date-type] input[type="text"]', this.form).each((i, el) => {
-      let $dateWrapper = $(el).parents('[data-provider="datepicker"]')
-      let initialDate = this.parseDate($dateWrapper, 'date')
-
+    $('[data-provider=datepicker]').each((i, el) => {
       $(el).datepicker({
```

Comment:
you use $(el) twice, you should put it in a variable

Reply:
fixed 38a02d0